### DISCUSSION TITLE
代码运行不起来

### DISCUSSION THREAD

**Discussion by qq547176052** [halo-dev/halo#2348] [General]
### 检查清单

- [X] 已经在 [GitHub Issues](https://github.com/halo-dev/halo/issues) 中搜索过该问题
- [X] 已经在[论坛](https://bbs.halo.run/)中搜索过该问题
- [X] 已经查阅过[文档](https://docs.halo.run/)

### 是什么版本出现了此问题？

编译失败

### 使用的什么数据库？

H2

### 使用的哪种方式部署？

Fat Jar

### 发生了什么？

我使用以下命令不能使halo博客运行
```
git clone --recursive https://github.com/halo-dev/halo
cd halo
gradlew.bat bootRun --args="--spring.profiles.active=dev --server.port=2333"
```
![image](https://user-images.githubusercontent.com/26636842/185999280-7bf12438-1c14-40a6-a6c8-18dd75c2dd66.png)

![image](https://user-images.githubusercontent.com/26636842/185998845-ba21d25f-4988-4b07-8a49-7b17bedc782f.png)

我知道 肯定是我有运行环境有问题 请你帮帮我 非常感谢

### 相关日志输出

```shell
运行不起来 没有日记
```

### 附加信息

![image](https://user-images.githubusercontent.com/26636842/185999539-66fb92b3-7012-4670-bf0c-3f15454d3ff6.png)

**Comment by JohnNiang** [comment]
日志截图不够完整，建议直接粘贴 `~/.halo/logs/spring.log` 到这里。

**Comment by ghost** [comment]
**需要将 jdk 版本设置为 11(2.0以上版本使用17)**
在 IDEA 中更改：
1. File -> Settings -> Build,Execution,Deployment -> Build Tools -> Gradle

![gradle-jdk11](https://user-images.githubusercontent.com/106857035/186072928-d65b09e7-971e-43f1-8e12-2f1b6020feb2.png)

2. File -> Project Structure -> Project
![project-jdk11](https://user-images.githubusercontent.com/106857035/186073010-713a31df-11fa-47e1-a24a-fe75ffe50490.png)

3. Run Debug -> Edit Configurations -> Build and run
![run-jdk11](https://user-images.githubusercontent.com/106857035/186073152-bb3d866c-0fc8-4eff-8cfc-afe0a5f6fe9b.png)

**Comment by jiale233** [comment]
一样的问题 解决了我的第一个bug  然后遇到了第二个bug 哭死

**Reply by JohnNiang** [reply]
遇到任何问题都可以提出来，尽可能详细，避免来回沟通。
